dynamic_db_router/router.py

```diff
@@ -83,45 +89,33 @@ def lowest_id_account():
     def __init__(self, database, read=True, write=False):
         self.read = read
         self.write = write
+        self.database = database
```

Comment:
What does this code do in the event the database is a string? Is there any reason to remove the check below? Or was this code only checked with database being a dictionary?

Reply:
If database is a string, the code assumes that it's a valid database alias already configured in Django’s DATABASES setting. No special handling is needed, we just use this string directly to reference the database. If database is a dictionary, we would need a more dynamic setup where you might not have a pre-configured database alias, and instead you would need to provide all necessary connection details. There isn't any reason to remove the check, since i was already defining the databases in the settings file, i was just focused on it working for strings. I'll add the check back in my next push

Reply:
Yes, let's add it back. The explicit exception is helpful. Given we now have python 3, we could also do `database: str | dict` to make use of Python's type hints.